packages/synapse-sdk/src/telemetry/adapters/sentry.ts

```diff
@@ -0,0 +1,170 @@
+import type { CustomEvent, HTTPEvent, OperationEvent, OperationType, TelemetryConfig } from '../types.ts'
+import { BaseTelemetryAdapter } from './base-adapter.ts'
+import { integrations, Sentry } from './sentry-dep.ts'
+
+/**
+ * Sentry telemetry adapter
+ * Works in both Node.js and browser - sentry-dep.js is swapped via package.json "browser" field
+ */
+export class SentryAdapter extends BaseTelemetryAdapter {
+  init(config: TelemetryConfig, tags: Record<string, string>): void {
+    Sentry.init({
+      dsn: 'https://3ed2ca5ff7067e58362dca65bcabd69c@o4510235322023936.ingest.us.sentry.io/4510235328184320',
+      // Setting this option to false will prevent the SDK from sending default PII data to Sentry.
+      // For example, automatic IP address collection on events
+      sendDefaultPii: false,
+      environment: config.environment || 'production',
+      beforeSend: this.sanitizeEvent,
+      // Enable tracing/performance monitoring
+      tracesSampleRate: 1.0, // Capture 100% of transactions for development (adjust in production)
+      // Integrations configured per-runtime in sentry-dep files
+      integrations,
+    })
+
+    Sentry.setContext('environment', {
+      sdkVersion: tags.sdkVersion,
+      runtime: tags.runtime,
+      network: tags.network,
+      ua: tags.ua,
+      appName: tags.appName,
+    })
+
+    // Set global tags
+    Sentry.setTag('sdk_version', tags.sdkVersion)
+    Sentry.setTag('runtime', tags.runtime)
+    Sentry.setTag('network', tags.network)
```

Comment:
Why do we have some of these as global, and some of them set to a context?

Reply:
sdk_version will allow us to search based off it, but we can also search based off of "release" which is now set, so i will clean this up.. only planning on `setTag` for now, and Context `environment` for things we don't care about searching but might be important when looking into breadcrumbs/events